taskgraph.json

```diff
@@ -25,7 +27,7 @@
             "gaia-tc-vcs":"/home/worker/.tc-vcs",
```

Comment:
For completeness we probably should add the other cache scopes to the higher level graph scopes I think.